dts/bindings/can/microchip,mcp251xfd.yaml

```
@@ -65,6 +65,10 @@ properties:
       is not set, then an internal clock (typically 40MHz or 20MHz) will be
       output on CLKO pin instead.
 
+  xstby-enable:
+    type: boolean
+    description: Enables standby pin control on GPIO0.
```

Suggestion to make the purpose of this pin more clear:

```suggestion
    description: Enables automatic transceiver standby pin control on GPIO0.
```